cfme/tests/pod/test_appliance_crud.py

```diff
@@ -121,6 +125,20 @@ def template_folder(template):
     return os.path.join(upload_folder, template['name'])
 
 
+@pytest.fixture(scope='function')
+def setup_ipa_auth_provider(temp_pod_appliance, ipa_auth_provider):
```

Comment:
**Question** Looks you have two fixtures doing the same thing?

Reply:
hehe, you're right. I apparently moved it to global fixtures file and forgot to remove it here

Reply:
nope, I was wrong both are necessary. since first fixture for regular appliance whereas second one overrides first one for custom appliance.